Makefile

```
@@ -1,24 +1,62 @@
-changelog:  # Install gh cli and jq first
+# Define default goal
+.DEFAULT_GOAL := help # Sets default action to be help
+# Define macro for printing help message
+define PRINT_HELP_PYSCRIPT # start of Python section
+import re, sys
+
+output = []
+# Loop through the lines in this file
+for line in sys.stdin:
+    # if the line has a command and a comment start with
+    #   two pound signs, add it to the output
+    match = re.match(r'^([a-zA-Z_-]+):.*?## (.*)$$', line)
+    if match:
+        target, help = match.groups()
+        output.append("%-10s %s" % (target, help))
+# Sort the output in alphanumeric order
+output.sort()
+# Print the help result
+print('\n'.join(output))
+endef
+export PRINT_HELP_PYSCRIPT # End of python section
+# Define target for printing help message
+help:
+	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
+# Define variable for GitHub API URL
+GH_API = https://api.github.com/repos/pydanny/listo/releases/latest
+# Define variable for jq command
+JQ = jq -r
+# Define target for generating changelog
+changelog:  ## Install gh cli and jq first
```

For the sake of clarity, add line breaks between sections.

```suggestion

changelog:  ## Install gh cli and jq first
```